sal_interfaces/SALSubsystems.xml

```diff
@@ -759,11 +759,11 @@
   <CSCDocs></CSCDocs>
   <ProductOwner>Brian Stalder</ProductOwner>
   <RelatedDocuments></RelatedDocuments>
-  <SoftwareLanguage></SoftwareLanguage>
+  <SoftwareLanguage>Python</SoftwareLanguage>
   <RuntimeLanguages>IDL</RuntimeLanguages>
   <VendorContact>DATControl</VendorContact>
-  <Simulator>No</Simulator>
-  <Configuration>Yes</Configuration>
+  <Simulator>Internal to CSC</Simulator>
+  <Configuration>Not yet</Configuration>
```

Comment:
This needs to be a guess at a URL. Making one up if there is none is OK for now. If HVAC is part of the EAS, I would use that GH URL.

Reply:
There is no CSC for this yet, which is why I set it to "Not yet". I changed it to "Not Configurable".